### ISSUE TITLE
API: Unable to create address.

### ISSUE THREAD

**Issue by by-tim** [swellstores/swell-js#27]
### Issue description
Unable to create a new address via `swell.account.createAddress`.

#### Steps to reproduce the issue

1.  Call using data from https://swell.store/docs/js/account/#create-an-address.
```
await swell.account.createAddress({
  name: "Julia Sanchez",
  address1: "Apartment 16B",
  address2: "2602 Pinewood Drive",
  city: "Jacksonville",
  state: "FL",
  zip: "32216",
  country: "United States",
  phone: "904-504-4760"
});
```
2. Log result.

#### What's the expected result?

- The object of the newly created address.

#### What's the actual result?

- Logged:
```
errors: {
  address1: {message: "Required", code: "REQUIRED"}
  country: {message: "Exceeds max length (2)", code: "LENGTH", length: 2} 
}
```

**Comment by ericingram** [comment]
@by-tim can you share the network request that produces this error?

The `country` error makes sense because it should be a 2-character country code

The `address1` error is odd since it looks like a straight pass through, and your code shows it being passed, so I can't tell why that one's happening.

**Comment by by-tim** [comment]
```
Request URL: https://test.swell.store/api/account/addresses
Request Method: POST
Status Code: 200 OK
Remote Address: 74.80.234.123:443
Referrer Policy: strict-origin-when-cross-origin
```

Request payload
```
address_1: "Apartment 16B"
address_2: "2602 Pinewood Drive"
city: "Jacksonville"
country: "US"
name: "Julia Sanchez"
phone: "904-504-4760"
state: "FL"
zip: "32216"
```

Response
```
{"errors":{"address1":{"message":"Required","code":"REQUIRED"}}}
```

**Comment by ericingram** [comment]
@by-tim this is now fixed in v3.4.2 (fixed `toSnake` method to avoid converting `address1` to `address_1`
